### ISSUE TITLE
`generic/fedora34` box is out of date 

### ISSUE THREAD

**Issue by gotmax23** [lavabit/robox#211]
Currently. there are 102 out of date packages in the fedora34 box. Is it possible to rebuild it?

Thank you for all the Vagrant boxes!
\- Maxwell

**Comment by timschumi** [comment]
The boxes should have been rebuilt with the 3.3.6 release that happened a week ago. Since those haven't been released on Vagrant Cloud yet (the usual build time appears to be around 2 days), I can only assume that the builds got stuck somewhere along the way...

**Comment by ladar** [comment]
Hi @gotmax23 ... @timschumi is correct. I rebuilt the boxes last week, but a few of the configs failed, which I managed to fix this week. The new boxes are uploading now, this should bring them inline with the updates as of late last week. 

I typically try and rebuild the boxes every 1-2 weeks, but that schedule can vary depending on my other commitments. As @timschumi mentioned, with the resources I have, a full build takes about 48 hours at this point, and that's assuming goes smoothly. Sometimes box configs will get stuck, which is what happened last week. That meant the libvirt boxes were only partially complete when I checked on Monday. If box configs do fail, they can hold up the release process till I have to investigate and troubleshoot.

I've wanted to hire an intern who can monitor the build process, improve the box config scripts, and add new configs, but don't have the resources right now. 

All of that said, I'm adding a Debian 11 box today, so I'll be kicking off a rebuild right away. Assuming all goes well, it should be ready to upload on Mon or Tues. 

If all your worried about is getting the package updates, might suggest adding `sudo dnf -y update` to your Vagrantfile? That will ensure your box is updated at provisioning time.
